projects/opendr_ws_2/src/data_generation/data_generation/synthetic_facial_generation.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3.6
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cv2
+import torch
+import rclpy
```

Comment:
For reasons explained in the next comment, `Node` should be imported. Also rclpy is missing from dependencies, see related comment in `package.xml`.

```suggestion
import rclpy
from rclpy.node import Node
```